shared/config/donation-settings.ts

```diff
@@ -41,7 +41,7 @@ export const donationSubscriptionConfig = {
     month: 'Monthly'
```

Comment:
```suggestion
    month: 'monthly'
```
There are a few cases where it's being lowercased (I got one, but the rest aren't touched by this PR). Could you update the rest?